### PR TITLE
[16.0][MIG] hr_expense_invoice: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

@Tecnativa TT45635

### CODE REVIEW

File: hr_expense_invoice/models/account_move.py [hunk 1]
```diff
@@ -26,3 +26,14 @@ def _check_expense_ids(self):
...
                     )
                 )
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    @api.constrains("account_id", "display_type")
+    def _check_payable_receivable(self):
+        items_with_expense = self.filtered(lambda x: x.expense_id)
```
Comment (chienandalu) [inline hr_expense_invoice/models/account_move.py]: nitpick:
```suggestion
        items_with_expense = self.filtered("expense_id")
```

**Review by chienandalu — APPROVED** [review]
Tested on runboat :+1:

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**